### PR TITLE
active user functional in database

### PR DESCRIPTION
**Pull request by sales-ls21**
# Description
Add description about what this pull request covers.
This connects the database and allows a user to be set as active or inactive. 

## Number of Fixes
Type of number of fixes included in this PR.
No fixes, all new code. 
## Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request
#5 
## Problem to Solve
Describe the problem(s) this pull request solves.
Users can be set to active. The code also only allows one user to be active at a time. 

## Proposed Changes
Describe the proposed changes.
No changes.
## Expected Behavior
Describe the expected behavior.
The active column in the customer table should update to reflect that the user passed is active. 
## Steps to Test Solution

1. git fetch 
2. git checkout this branch
3. navigate to app folder
4. run python customerstatusmanager.py
5. open dbrowser and check customer table. The user passed should have an active attribute that equals 1 and all others should have active equal 0.

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

### CODE REVIEW

File: app/customerstatusmanager.py [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
\ No newline at end of file
+            try:
+                c.execute("UPDATE Customers SET active=0 WHERE active=1")
+                c.execute("UPDATE Customers SET active=1 WHERE customer_name = '{}'".format(user))
+            except sqlite3.OperationalError:
+                return "There was an error. Please try again."
+          
+if __name__ == '__main__':
+    activeUser = CustomerStatusManager()
+    activeUser.change_status("John Doe")
```
Comment (stevenwally) [inline app/customerstatusmanager.py]: We may need to refactor this line in the future to accommodate all users and not just John Doe.

**Comment by nathantbaker** [comment]
The logic of the method is working perfectly. If the method takes customer name as the argument, you may want to more explicitly name that argument something like customer_name so other parts of the app know what to pass it.

Also, the test for it is failing because you're passing a customer object to the test and just the customer name to the method, so you'll either want to pass just the name in the test or refactor the method to take a customer object.

**Comment by nathantbaker** [comment]
👍  looks good. one test isn't passing as expected because we are refactoring all tests to instantiate test users with street addresses where they all don't currently.

**Review by stevenwally — APPROVED** [review]

**Review by stevenwally — APPROVED** [review]
👍 

fetched and tested branch. 

works as described - echoing nathan's note.

**Pull request merged**